### ISSUE TITLE
Can't see and play Musik with non-standard letters

### ISSUE THREAD

**Issue by DonMartio** [strawberrymusicplayer/strawberry#855]
I can't see and play music from Artists like Antonín Dvořák.

Is this an utf8 issue?

**Comment by jonaski** [comment]
Hard to tell from the info you give, there isn't any known UTF-8 issue that I know of. By not seeing the music, do you mean that the songs do not appear in the collection?

**Comment by DonMartio** [comment]
Jepp i can't even drop them manually into strawberry.
I get kind of this Error Message if i try to:
`File /Musik/Antonín Dvořák/Witold Rowicki/Symphony No 9/01 - I. Adagio — Allegro molto.mp3 does not exist.`

**Comment by DonMartio** [comment]
Ok, i copied the stuff to Antonin Dvorak and i got the same error with the mentioned piece of music but not with the second on of the symphony. So i guess it is just an issue with the Directory/Filenames (01 - I. Adagio — Allegro molto.mp3 contains the long hyphen).

**Comment by jonaski** [comment]
I can't reproduce any issues with those characters, this is probably related to system setup or Qt version.

**Comment by DonMartio** [comment]
Thank you for your reply.

Just for the record. I did some research (build small Test-Apps in C++ and Python, changed and compiled strawberry from the sources) to figure out what the Problem is. 
I got rid of my problem by adding `QTextCodec::setCodecForLocale(QTextCodec::codecForName("UTF-8"))`
to the main.cpp which brought me on the right track.
I had to remove the LC_ALL="" entry from my pretty old /etc/env.d/02locale.
Just in case somebody else struggles with this.
